src/Microsoft.Azure.SignalR.Management/IHubLifetimeManagerForUserGroup.cs

```diff
@@ -10,13 +10,15 @@ namespace Microsoft.Azure.SignalR.Management
     internal interface IHubLifetimeManagerForUserGroup
     {
         Task UserAddToGroupAsync(string userId, string groupName, CancellationToken cancellationToken = default);
-        
+
         Task UserAddToGroupAsync(string userId, string groupName, TimeSpan ttl, CancellationToken cancellationToken = default);
 
         Task UserRemoveFromGroupAsync(string userId, string groupName, CancellationToken cancellationToken = default);
 
         Task UserRemoveFromAllGroupsAsync(string userId, CancellationToken cancellationToken = default);
 
         Task<bool> IsUserInGroup(string userId, string groupName, CancellationToken cancellationToken = default);
+
+        Task DisposeAsync();
```

Comment:
rename interface? not only for user group

Reply:
or split them, put DisposeAsync to another interface

Reply:
@vicancy what do you think?

Reply:
I think it's a little overdesigned to define the method in another interface?